src/pipert2/utils/method_data.py

```diff
@@ -8,5 +10,19 @@
 
 @dataclass
 class Method:
-    name: str
+    event_name: str
+    routines_by_flow: Dict[str, Optional[list]] = None
```

Comment:
What do you say that the default value of this data member will be a dict of lists ?

Reply:
You right, I think it's better to use default dict because of the next situation - 
let's say someone put values in this dict and then delete them, we run nothing instead run everything as expected if there are not specific flows and routines.
It will be changed